packages/rpc-transformers/src/params-transformer.ts

```diff
@@ -28,23 +28,11 @@ export function getDefaultRequestTransformerForSolanaRpc(config?: RequestTransfo
             keyPath: [],
         };
         const patchedRequest = { methodName, params: traverse(rawParams, initialState) as T };
-        if (!Array.isArray(patchedRequest.params)) {
-            return patchedRequest;
-        }
-        const optionsObjectPositionInParams = OPTIONS_OBJECT_POSITION_BY_METHOD[methodName];
-        if (optionsObjectPositionInParams == null) {
-            return patchedRequest;
-        }
         return pipe(
             patchedRequest,
-            ({ methodName, params }) => ({
-                methodName,
-                params: applyDefaultCommitment({
-                    commitmentPropertyName: methodName === 'sendTransaction' ? 'preflightCommitment' : 'commitment',
-                    optionsObjectPositionInParams,
-                    overrideCommitment: defaultCommitment,
-                    params: params as unknown[],
-                }),
+            getDefaultCommitmentRequestTransformer({
+                defaultCommitment,
+                optionsObjectPositionByMethod: OPTIONS_OBJECT_POSITION_BY_METHOD,
```

Comment:
Sweet! So someone could also use `pipe(...)` to create a custom request transformer that builds on top of `getDefaultCommitmentRequestTransformer`.